ogc/src/main/scala/geotrellis/server/ogc/utils/Implicits.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.server.ogc.utils
+
+import scalaxb.DataRecord
+
+import scala.xml.Elem
+
+trait Implicits {
+  implicit class DataRecordMethods[A](dataRecord: DataRecord[A]) {
```

Comment:
Does this make the IntelliJ error go away or does it address something else?

Reply:
@CloudNiner just syntax `DataRecord("all").toXML` looks better than `ScalaxbUtils.toXML(DataRecord("all"))`